AnsiC.gitattributes

```diff
@@ -0,0 +1,30 @@
+# Handle line endings automatically for files detected as text
+# and leave all files detected as binary untouched.
+* text=auto
+
+#
+# The above will handle all files NOT found below
+#
+# These files are text and should be normalized (Convert crlf => lf)
+*.c             text diff=c
+*.h             text diff=c
+*.json          text
+*.rc            text
+*.sh            text
```

Comment:
`eol=lf` should be used for `*.sh` files.

Reply:
corrected